.gitattributes

```diff
@@ -0,0 +1,2 @@
+* text=auto
+*.sh eol=lf
```

Comment:
By default of Git for Windows, it converts LF to CRLF. see also https://help.github.com/articles/dealing-with-line-endings/#per-repository-settings